pyaerocom/varnameinfo.py

```diff
@@ -134,8 +134,9 @@ def __str__(self):
         s = (
             f"\nVariable {self.var_name}\n"
             f"is_wavelength_dependent: {self.is_wavelength_dependent}\n"
-            f"is_optical_density: {self.is_optical_density}"
         )
+        if hasattr(self, "is_optical_density"):
+            s += f"is_optical_density: {self.is_optical_density}\n"  # pragma: no cover # can't find situation where this happens however not sure if depricated
```

Comment:
the `pragma: no cover` should be on the `if` line
``` python
        if hasattr(self, "is_optical_density"): # pragma: no cover 
            s += f"is_optical_density: {self.is_optical_density}\n"  # can't find situation where this happens however not sure if deprecated
```
that way it is clear that the whole if branch wont be covered